stubs/xtea/xtea/__init__.pyi

```diff
@@ -0,0 +1,25 @@
+from collections.abc import Callable
+
+from pep272_encryption import PEP272Cipher
+
+MODE_ECB: int
+MODE_CBC: int
+MODE_CFB: int
+MODE_PGP: int
+MODE_OFB: int
+MODE_CTR: int
+block_size: int
+key_size: int
+
+def new(key, **kwargs): ...
+
+class XTEACipher(PEP272Cipher):
+    block_size: int
+    IV: bytes | None
+    counter: Callable[[], bytes] | None
+    rounds: float | None
+    cycles: float | None
+    endian: str | None
+    def __init__(self, key: bytes, mode: int | None = ..., **kwargs) -> None: ...
+    def encrypt_block(self, key: bytes, block, **kwargs): ...
+    def decrypt_block(self, key: bytes, block, **kwargs): ...
```

Comment:
```suggestion
    def __init__(self, key: ReadableBuffer, mode: int | None = ..., **kwargs) -> None: ...
    def encrypt_block(self, key: ReadableBuffer, block: ReadableBuffer, **kwargs): ...
    def decrypt_block(self, key: ReadableBuffer, block: ReadableBuffer, **kwargs): ...
```
These are passed to `struct.unpack` which accepts any `ReadableBuffer`. (Though in fact the two methods appear to ignore the `key` parameter.)